singlePokemon.js

```diff
@@ -0,0 +1,14 @@
+import displayPokemon from './src/displaySinglePokemon.js';
+import fetchPokemon from './src/fetchPokemon.js';
+
+const presentPokemon = async () => {
```

Comment:
I think you are trying to render the pokemon in this function, so a better name would be "renderPokemon"